addOns/packpentester/src/main/javahelp/help/map.jhm

```diff
@@ -0,0 +1,9 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<!DOCTYPE map
+  PUBLIC "-//Sun Microsystems Inc.//DTD JavaHelp Map Version 1.0//EN"
+         "http://java.sun.com/products/javahelp/map_1_0.dtd">
+
+<map version="1.0">
+    <mapID target="packpentester" url="contents/packpentester.html" />
+    <mapID target="box-pen-icon" url="contents/images/box--pencil.png" />
```

Comment:
I don't know if it's worth caring but box-pencil/box-pen consistency? (In toc.xml too)